src/goBuild.ts

```diff
@@ -56,7 +56,7 @@ export function goBuild(fileUri: vscode.Uri, goConfig: vscode.WorkspaceConfigura
 	}
 
 	const buildEnv = Object.assign({}, getToolsEnvVars());
-	const tmpPath = path.normalize(path.join(os.tmpdir(), 'go-code-check'));
+	const tmpPath = path.normalize(path.join(os.tmpdir(), 'go-code-check.' + os.userInfo().username));

```

Comment:
Can we hash that name? There are cases where when trying to investigate an issue I request users to paste a screenshot of the output channel. User name is clearly PII 100% of the time and I want to stay clear of it.

Also, if we hash lets hash it once and re-use it.

Reply:
It doesn't really even need be a username, the filename could be anything providing it didn't cause a collision with another user.

I guess a hash based off username would be okay though, saves randomizing the filename and filling up /tmp.

I'll take another look at it 👍